src/modeler/AutoRest.Swagger/Validation/CollectionObjectPropertiesNamingValidation.cs

```diff
@@ -26,6 +26,11 @@ public class CollectionObjectPropertiesNamingValidation : TypedRule<Dictionary<s
         /// </summary>
         public override ValidationCategory ValidationCategory => ValidationCategory.RPCViolation;
 
+        /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
I believe renaming a model's property that is readOnly means stuff needs to change at the service end.